README.md

```diff
@@ -6,65 +10,77 @@
 
 ## Serverless Web Archive Replay
```

Comment:
Replay is in the name already, maybe give new folks some more adjectives to think about when understanding what the app does?  Alternatively, this heading could just be removed.

```suggestion
## Serverless Web Archive Viewer
```